internal/app/activityvisitassociate/activityVisitAssociate.go

```diff
@@ -0,0 +1,80 @@
+package activityvisitassociate
```

Comment:
Name of the package is too long. better use a shorter name.
Name of the file doesn't follow our convention, please correct. Fix the same for other files